code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -11,11 +10,11 @@
 			return
 		if(client?.prefs?.preferred_slot)
 			if(draw_from_slot_if_possible(client.prefs.preferred_slot))
-				next_move = world.time + 3
+				next_move = world.time
 				return
 		for(var/slot in SLOT_DRAW_ORDER)
 			if(draw_from_slot_if_possible(slot))
-				next_move = world.time + 3
+				next_move = world.time
```

Comment:
what is this

Reply:
well removing the delay that was in drawing weapons #5807

Reply:
nonatomized keep this